render/topologies_test.go

```diff
@@ -412,17 +412,17 @@ func TestContainerRenderer(t *testing.T) {
 			LabelMinor: serverHostName,
 			Rank:       serverContainerImageID,
 			Pseudo:     false,
-			Adjacency:  report.MakeIDList(clientContainerID, render.UncontainedID, render.TheInternetID),
+			Adjacency:  report.MakeIDList(clientContainerID, render.TheInternetID),
 			Origins:    report.MakeIDList(serverContainerNodeID, server80NodeID, serverProcessNodeID, serverHostNodeID),
 			AggregateMetadata: render.AggregateMetadata{
 				render.KeyBytesIngress: 150,
 				render.KeyBytesEgress:  1500,
 			},
 		},
-		render.UncontainedID: {
-			ID:                render.UncontainedID,
+		fmt.Sprintf("%s:%s", render.UncontainedID, serverHostName): {
+			ID:                fmt.Sprintf("%s:%s", render.UncontainedID, serverHostName),
```

Comment:
Ugh. I see. There is some prior art in fmt.Sprintfing renderable node IDs. Gross. OK, continue as-is, we'll address later :(